source/external/cJSON.c

```diff
@@ -86,87 +86,93 @@ THE SOFTWARE.
 #endif
 
 typedef struct {
-  const unsigned char *json;
-  size_t position;
+    const unsigned char *json;
+    size_t position;
 } error;
+static error global_error = { NULL, 0 };
+
+CJSON_PUBLIC(const char *) cJSON_GetErrorPtr(void)
+{
+    return (const char*) (global_error.json + global_error.position);
+}
```

Comment:
global_error and `cJSON_GetErrorPtr` is removed in our copy of cJson due to race conditions [#972 (files)](https://github.com/awslabs/aws-c-common/pull/972/files)